epub33/core/index.html

```diff
@@ -1502,6 +1502,19 @@ <h3>Data URLs</h3>
 						restrictions</a>).</p>
 			</section>
 
+			<section id="sec-file-urls">
+				<h3>File URLs</h3>
+				<p>
+					The <a data-cite="rfc8089#"><code>file:</code> URL scheme</a> is defined in [[rfc8089]] as "identifying an 
+					object (a 'file') stored in a structured object naming and accessing environment on a host (a 'file system')."
+					It is typically used to retrieve files from within one's own computer.
+				</p>
+				<p>
+					Using a file URL in an [=EPUB publication=], which can be transferred among different hosts, represents a security risk.
```

Comment:
It's also just plain non-interoperable. What file could possibly exist on all reading systems with the exact same path and name?

Reply:
Added that, too. I leave the remark on security, it does not harm...